src/azure-cli/azure/cli/command_modules/servicefabric/tests/latest/test_sf_managed_cluster.py

```diff
@@ -123,7 +124,7 @@ def test_node_type_operation(self):
         timeout = time.time() + 300
         while True:
             try:
-                self.cmd('az sf managed-node-type node restart -g {rg} -c {cluster_name} -n snt --node-name snt_0 snt_1')
+                self.cmd('az sf managed-node-type node restart -g {rg} -c {cluster_name} -n snt --node-name snt_0 snt_1 --update-type {update_type}')
```

Comment:
probably better to create a new test for byUpgradeDomain case and removing --node-name. Usually when customers want to run with that flag they want to do it for the whole node type

Reply:
updated